tests/src/test/java/alluxio/server/ft/journal/MultiMasterJournalTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.server.ft.journal;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.AlluxioURI;
+import alluxio.Configuration;
+import alluxio.Constants;
+import alluxio.PropertyKey;
+import alluxio.master.MultiMasterLocalAlluxioCluster;
+import alluxio.master.NoopMaster;
+import alluxio.master.journal.JournalUtils;
+import alluxio.master.journal.ufs.UfsJournal;
+import alluxio.master.journal.ufs.UfsJournalSnapshot;
+import alluxio.util.CommonUtils;
+import alluxio.util.URIUtils;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class MultiMasterJournalTest {
+  private MultiMasterLocalAlluxioCluster mCluster;
+
+  @Before
+  public void before() throws Exception {
+    mCluster = new MultiMasterLocalAlluxioCluster(2, 2);
```

Comment:
why 2 workers?

Reply:
2 is an arbitrary choice, the original choice of 0 worker did not work because `createFile` failed with `No available Alluxio worker found`